nxdrive/manager.py

```diff
@@ -508,6 +510,24 @@ def set_direct_edit_auto_lock(self, value: bool) -> None:
         log.debug(f"Changed parameter 'direct_edit_auto_lock' to {value}")
         self.dao.store_bool("direct_edit_auto_lock", value)
 
+    @pyqtSlot(str, result=bool)
+    def get_feature_state(self, name: str) -> bool:
+        return Feature.__dict__[name]
```

Comment:
```suggestion
        return getattr(Feature, name)
```
- Missing docstring.